autoload/db_ui/dbout.vim

```diff
@@ -127,6 +128,33 @@ function! db_ui#dbout#yank_header() abort
 endfunction
 
 function! s:get_cell_range(cell_line_number, curpos, scheme_name) abort
+  if a:scheme_name ==# 'oracle'
+    return s:get_virtual_cell_range(a:cell_line_number, a:curpos)
+  endif
+
+  let line = getline(a:cell_line_number)
+  let table_line = '-'
+
+  let col = a:curpos[2] - 1
```

Comment:
I kept `getcurpos()`, only because `getcurpos()[2]` is the same as `col()`. That way we can make these two functions play together more nicely

Reply:
Ok, I think we should be ok.